gammapy/data/gti.py

```diff
@@ -318,6 +318,35 @@ def select_time(self, time_interval):
 
         return self.__class__(gti_within)
 
+    def delete_interval(self, time_interval):
+        """Select and crop GTIs in time interval.
+
+        Parameters
+        ----------
+        time_interval : `astropy.time.Time`
```

Comment:
The docstring says that this is an interval, but the type is just one Time. Should not it be a [Time, Time]?